src/test/java/in/zapr/druid/druidry/query/aggregation/GroupByTest.java

```diff
@@ -92,8 +92,8 @@ public void testSampleQuery() throws JsonProcessingException, JSONException {
                 "    ]\n" +
                 "  },\n" +
                 "  \"aggregations\": [\n" +
-                "    { \"type\": \"longSum\", \"name\": \"total_usage\", \"fieldName\": \"user_count\" },\n" +
-                "    { \"type\": \"doubleSum\", \"name\": \"data_transfer\", \"fieldName\": \"data_transfer\" }\n" +
+                "    { \"type\": \"longSum\", \"name\": \"total_usage\", \"fieldName\": \"user_count\", \"expression\": null },\n" +
```

Comment:
these setting of explicit `null`s would not be required after `JsonInclude.Include.NON_NULL`